src/components/wizard-navigation-bar.component.ts

```diff
@@ -116,7 +127,7 @@ export class WizardNavigationBarComponent {
    * @returns True if the step can be marked as optional
    */
   public isOptional(wizardStep: WizardStep): boolean {
-    return wizardStep.optional && !wizardStep.completed && !wizardStep.selected && !this.wizardState.completed
+    return wizardStep.optional
```

Comment:
Please add a `;` to the end of this line.

Reply:
I will.
In my defense, there was none before